### PR TITLE
Add config command to cli

### PR DESCRIPTION
**Pull request by xicombd**
Example uses:
- `jsipfs config foo` - get the foo value
- `jsipfs config foo bar` - set foo as `'bar'`
- `jsipfs config foo true --bool` - set foo as `true`
- `jsipfs config foo '{"bar: "0"}' --json` - set foo as `{bar: 0}`

Related to #49

### CODE REVIEW

File: tests/test-cli/test-config.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    nexpect.spawn('node', [process.cwd() + '/src/cli/bin.js', 'config', 'foo', '{"bar: 0}', '--json'])
+      .run((err, stdout, exitcode) => {
+        const expected = 'error	invalid JSON provided'
+        expect(stdout[0]).to.equal(expected)
+        expect(err).to.not.exist
+        expect(exitcode).to.equal(1)
+        done()
+      })
+  })
+})
```
Comment (Dignifiedquire) [inline tests/test-cli/test-config.js]: Missing a test for failure, i.e. no `key` present

Comment (xicombd) [inline tests/test-cli/test-config.js]: Yup, just added it here: https://github.com/xicombd/js-ipfs/commit/ebf25ef27b10593858f395bd6a17264558f4836f#diff-1a445abf8db16914970d37a823fd6e7eR77

**Comment by Dignifiedquire** [comment]
Thanks :octocat: LGTM, but I'll let @diasdavid look at it before doing anything

**Comment by diasdavid** [comment]
Thank you Francisco :D great you had the time to finish the PR. I'll be hacking tomorrow the whole afternoon again at the same place, join us :)

**Pull request merged**